myhpi/static/scss/footer.scss

```diff
@@ -1,5 +1,5 @@
 .footer-myhpi {
-    background-color: var(--bs-gray-700);
```

Comment:
The breadcrumbs chevrolets could be white

![grafik](https://github.com/user-attachments/assets/d15e8311-7a1a-40aa-8f0a-5218d5d4bdbc)